src/lib.rs

```diff
@@ -27,9 +27,17 @@
     )
 )]
 
+#![cfg_attr(not(feature = "std"), no_std)]
+
```

Comment:
This one is complex to read, please add a comment. Do we need the blank line? (If it applies only to the byteorder crate, probably not?)